dataactvalidator/filestreaming/loadFile.py

```diff
@@ -1,53 +1,216 @@
 import os
+import pandas as pd
+import boto
+import glob
 from dataactvalidator.filestreaming.loaderUtils import LoaderUtils
 from dataactvalidator.interfaces.validatorValidationInterface import ValidatorValidationInterface
-from dataactvalidator.filestreaming.csvS3Reader import CsvS3Reader
 from dataactcore.models.domainModels import CGAC,ObjectClass,ProgramActivity,SF133
 from dataactcore.config import CONFIG_BROKER
+from dataactvalidator.filestreaming.csvS3Reader import CsvS3Reader
+from sqlalchemy import and_
+
 
 def loadCgac(filename):
-    LoaderUtils.loadCsv(filename,CGAC,ValidatorValidationInterface(),{"cgac":"cgac_code","agency":"agency_name"},{"cgac_code":{"pad_to_length":3,"skip_duplicates":True}})
+    interface = ValidatorValidationInterface()
+    model = CGAC
+
+    # for CGAC, delete and replace values
+    interface.session.query(model).delete()
+    interface.session.commit()
+
+    # read CGAC values from csv
+    data = pd.read_csv(filename, dtype=str)
+    # toss out rows with missing CGAC codes
+    data = data[data['CGAC'].notnull()]
+    # clean data
+    data = LoaderUtils.cleanData(
+        data,
+        model,
+        {"cgac": "cgac_code", "agency": "agency_name"},
+        {"cgac_code": {"pad_to_length": 3}}
+    )
+    # de-dupe
+    data.drop_duplicates(subset=['cgac_code'], inplace=True)
+    # insert to db
+    LoaderUtils.insertDataframe(data, model.__table__.name, interface.engine)
+
 
 def loadObjectClass(filename):
-    LoaderUtils.loadCsv(filename,ObjectClass,ValidatorValidationInterface(),{"max_oc_code":"object_class_code","max_object_class_name":"object_class_name"},{"object_class_code":{"skip_duplicates":True}})
+    interface = ValidatorValidationInterface()
+    model = ObjectClass
+
+    # for object class, delete and replace values
+    interface.session.query(model).delete()
+    interface.session.commit()
+
+    data = pd.read_csv(filename, dtype=str)
+    # toss out blank rows
+    data.dropna(inplace=True)
```

Comment:
It might be nice to document why we're using `dropna` here, `notnull` elsewhere, and neither in other places.